src/wdio/services/TerraCommands/match-screenshot-and-test-standards.js

```diff
@@ -0,0 +1,43 @@
+import accessibilityMethods from './accessiblity';
+import visualRegressionMethods from './visual-regression';
+
+const determineOptions = (...args) => {
+  const param1 = args.length ? args[0] : undefined;
+  const param2 = args.length > 1 ? args[1] : undefined;
+
+  let name = 'default';
+  let options = {};
+  if (typeof param1 === 'string') {
+    name = param1;
+    options = typeof param2 === 'object' && !Array.isArray(param2) ? param2 : options;
+  } else {
+    options = typeof param1 === 'object' && !Array.isArray(param1) ? param1 : options;
+  }
+
+  // Check if custom selector should be used, otherwise use the global value.
+  const selector = options.selector || global.browser.options.terra.selector;
+
+  // Check if custom misMatchTolerance should be used, otherwise use the global value.
+  const misMatchTolerance = options.misMatchTolerance || global.browser.options.visualRegression.compare.misMatchTolerance;
```

Comment:
This is a new piece. Since this is the direction we are pushing people, should we place a misMatchTolerance maximum on screenshot comparison?

Reply:
What are people's thoughts on this?  I'm not sure what we'd want the max to be.  Any thoughts?

Reply:
A diff more the 1.0 is usually noticeable where I've found diffs below 1 to not be noticeable unless you specifically know what you're looking for.